packages/logging-interceptor/src/log.decorator.ts

```diff
@@ -28,6 +28,10 @@ export interface MaskingOptions {
    * If it is an array of strings, it will mask only the specified fields.
    */
   response?: string[] | boolean;
+  /**
+   * If true, it will disable the header masking.
+   */
+  disableHeaderMask?: boolean;
```

Comment:
It already exists an option `disableMasking` at the interceptor level. So, to be consistent, it would be better:
```suggestion
  disableHeaderMasking?: boolean;
```